appveyor.yml

```diff
@@ -57,7 +57,7 @@ test_script:
 after_test:
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, ubuntu18]"
   - rake "create_linux_build[%APPVEYOR_BUILD_VERSION%, %APPVEYOR_BUILD_FOLDER%, centOS7]"
-  - ps: copy ospsuite_*.zip ospsuite_windows.zip
+  - ps: copy ospsuite_*.zip ospsuite.zip

```

Comment:
@msevestre 
needed to rename the zip file, otherwise it cannot be installed under windows
(under linux no problem, if the .tar.gz name differs from the package name, but under windows apparently it is)

Reply:
I knew I  had tried this before
and ospsuite_windows...you can install it on windows? WHAT